lib/index.ts

```diff
@@ -131,6 +130,44 @@ class LogService {
     }
   }
 
+  addLogger(
+    loggerScopeFunction: (config: InitialConfig) => LoggerPlugin,
+    pluginConfig: PluginConfig) {
+    if (typeof loggerScopeFunction !== 'function')
+      return;
+
+    const loggerOptions = loggerScopeFunction.call(this, this.globalConfig);
```

Comment:
pls rename `loggerOptions` to `loggerPlugin` since `loggerScopeFunction` returns a `LoggerPlugin`.